src/main/java/com/gmail/visualbukkit/blocks/StatementBlock.java

```diff
@@ -79,9 +80,10 @@ public StatementBlock() {
                 Dragboard dragboard = startDragAndDrop(TransferMode.ANY);
                 SnapshotParameters snapshotParameters = new SnapshotParameters();
                 snapshotParameters.setFill(Color.TRANSPARENT);
-                dragboard.setDragView(snapshot(snapshotParameters, null));
+                dragboard.setDragView(snapshot(snapshotParameters, null), e.getX(), e.getY());
                 ClipboardContent content = new ClipboardContent();
                 content.putString("");
```

Comment:
Remove

Reply:
This is needed to keep it looking good on the cursor.